### ISSUE TITLE
Replacing the "From" operator logic

### ISSUE THREAD

**Issue by YeastCell** [CancerDataAggregator/cda-python#70]
The `From` operator was introduced to enable including multiple values of the same field into query criteria.
The most illustrative example is querying the patients that have data coming from both GDC and PDC data nodes.
One of the problems of the current implementation is that it utilizes subqueries to resolve the `From` operator which then complicates the logic for the files endpoint.

The following is the suggestion for replacing the `From` operator such that we achieve the same results by adding conditions into a single where clause instead of creating subqueries.

cda-python code:

```
q1 = Q('File.identifier.system = ("GDC", "IDC")')
q2 = Q('sex = "female"')

q = q1.And(q2)
r = q.run(table='gdc-bq-sample.R2_practice_kit', version='all_v1', select='Patient')
```

SQL translation:

```
SELECT nodes.id,
       ANY_VALUE(nodes.sex) as sex,
       ANY_VALUE(nodes.race) as race,
       ANY_VALUE(nodes.ethnicity) as ethnicity,
       ANY_VALUE(nodes.days_to_birth) as days_to_birth,
       ANY_VALUE(nodes.File) as File,
       ANY_VALUE(nodes.ResearchSubject) as ResearchSubject
FROM `gdc-bq-sample.R2_practice_kit.all_v1` AS nodes,
UNNEST(File) AS _File,
UNNEST(identifier) AS _identifier
WHERE (
    nodes.id IN (
        SELECT nodes.id
        FROM `gdc-bq-sample.R2_practice_kit.all_v1` AS nodes,
        UNNEST(File) AS _File,
        UNNEST(identifier) AS _identifier
        WHERE _identifier.system = "GDC"
        GROUP BY nodes.id
        HAVING COUNT(*) >= 1
    )
    AND
    nodes.id IN (
        SELECT nodes.id
        FROM `gdc-bq-sample.R2_practice_kit.all_v1` AS nodes,
        UNNEST(File) AS _File,
        UNNEST(identifier) AS _identifier
        WHERE _identifier.system = "IDC"
        GROUP BY nodes.id
        HAVING COUNT(*) >= 1
    )
    AND
    nodes.sex = "female"
)
GROUP BY nodes.id
```

The assignment syntax in cda-python code is just a suggestion, and perhaps should be decided based on developers' inputs.
The translated BQ code shows how the practice tables are queried (it's good for testing because we know what results to expect), but the only thing that would be different with production tables is the `SELECT` clause. This first needs to be resolved by https://github.com/CancerDataAggregator/cda-service/issues/69. 

To sum up – this only represents the replacement for the `From` operator and it doesn't yet provide the complete files endpoint logic. Though, it sets up the majority of elements that are needed for supporting the files endpoint.

**Comment by ACharbonneau** [comment]
Lets decide if we're keeping FROM or not @rfricke-asymmetrik

**Comment by ACharbonneau** [comment]
I think we decided that we have already made a better solution on accident
